tests/CLI_test/conftest.py

```diff
@@ -0,0 +1,60 @@
+import os
+import sys
+from pathlib import Path
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+from cmd2.utils import StdSim
+from loguru import logger
+
+from FABulous.FABulous_CLI.FABulous_CLI import FABulous_CLI
+from FABulous.FABulous_CLI.helper import create_project, setup_logger
+
+
+def normalize(block: str):
+    """Normalize a block of text to perform comparison.
+
+    Strip newlines from the very beginning and very end  Then split into separate lines and strip trailing whitespace
+    from each line.
+    """
+    assert isinstance(block, str)
+    block = block.strip("\n")
+    return [line.rstrip() for line in block.splitlines()]
+
+
+def run_cmd(app, cmd):
+    """Clear out and err StdSim buffers, run the command, and return out and err"""
+    app.onecmd_plus_hooks(cmd)
+
+
+TILE = "LUT4AB"
```

Comment:
This should be parameterizeable for the tests.  Maybe as a fixture?

Reply:
It is more like a constant, since it can be any tile. When we test for detail fabric generation with things like cocotb, then generating which tile will become important then we might want it to be a parameterizeable fixture.

Reply:
Sure, for now it's good :)